src/auditor/cli.py

```diff
@@ -38,24 +43,66 @@ def cli():
+            #: Only dump summary info to the console if verbose
+            #: Note: currently, org-wide checks (duplicate names) only reported if verbose
+            if args['--verbose']:
+                cli_handler = logging.StreamHandler(stream=sys.stdout)
+                cli_handler.setLevel(logging.DEBUG)
+                cli_formatter = logging.Formatter(
+                    fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
+                )
+                cli_handler.setFormatter(cli_formatter)
+                summary_logger.addHandler(cli_handler)
 
-        if args['--dry']:
-            org_auditor.check_items(args['--save_report'])
-        else:
-            org_auditor.check_items(report=False)  #: only do the fix report on a full run.
-            org_auditor.fix_items(args['--save_report'])
+            #: Set up org, check & fix items
+            org_auditor = Auditor(summary_logger, args['--verbose'], args['ITEM'])
+            if args['--dry']:
+                org_auditor.check_items(args['--save_report'])
+            else:
+                org_auditor.check_items(report=False)  #: only do the fix report on a full run.
+                org_auditor.fix_items(args['--save_report'])
+            org_auditor.check_organization_wide()
+
+        elif args['scheduled']:
+
+            #: Create a string stream for summary report
+            summary_stream = StringIO()
+            summary_handler = logging.StreamHandler(stream=summary_stream)
+            stream_formatter = logging.Formatter(
+                fmt='<pre>%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s</pre>',
+                datefmt='%m-%d %H:%M:%S'
+            )
+            summary_handler.setFormatter(stream_formatter)
+            summary_logger.addHandler(summary_handler)
 
-        org_auditor.check_organization_wide()
+            #: set up supervisor, add email handler
+            auditor_supervisor = Supervisor(
+                project_name='auditor', logger=summary_logger, log_path=credentials.REPORT_BASE_PATH
+            )
+            auditor_supervisor.add_message_handler(EmailHandler(credentials.EMAIL_SETTINGS))
+
+            #: Set up org, check & fix items
+            org_auditor = Auditor(summary_logger)
+            org_auditor.check_organization_wide()
+            org_auditor.check_items(report=False)  #: Checks will be reported in fix report
+            org_auditor.fix_items(report=True)
+
+            #: Build and send summary message
+            summary_message = MessageDetails()
+            summary_message.message = summary_stream.getvalue()
+            summary_message.project_name = 'auditor'
+            summary_message.attachments = [credentials.REPORT_BASE_PATH]
+            summary_message.subject = f'Auditor Report {datetime.datetime.today()}'
+
+            auditor_supervisor.notify(summary_message)
+
+        else:
```

Comment:
**nitpick**: this branching is not necessary and the final `else` is unreachable. docopt won't allow the code to execute if a command isn't provided or it doesn't match the doc string. you can remove the `elif` and `else` and return from the first commands if.

Reply:
Thanks for the pointer. I felt dirty doing an if/elif/else but couldn't wrap my head around how to avoid it that day.